Sources/Apollo/ApolloStore.swift

```diff
@@ -44,6 +47,9 @@ public final class ApolloStore {
     }
   }
 
+  /// Clears ths instance of the cache. Note that a cache can be shared across multiple `ApolloClient` objects, so clearing that underlying cache will clear it for all clients.
```

Comment:
Typo :)
```suggestion
  /// Clears the instance of the cache. Note that a cache can be shared across multiple `ApolloClient` objects, so clearing that underlying cache will clear it for all clients.
```

Reply:
English may be my first language but typo is clearly my second 🤦‍♀️